modules/react/common/lib/CanvasProvider.tsx

```diff
@@ -71,22 +65,20 @@ export const useCanvasThemeToCssVars = (
   const className = (elemProps.className || '').split(' ').concat(defaultBranding).join(' ');
   const style = elemProps.style || {};
   const {palette} = filledTheme.canvas;
-  (['common', 'primary', 'error', 'alert', 'success', 'neutral', 'action'] as const).forEach(
-    color => {
-      if (color === 'common') {
+  (['common', 'primary', 'error', 'alert', 'success', 'neutral'] as const).forEach(color => {
+    if (color === 'common') {
+      // @ts-ignore
```

Comment:
What is TS complaining about here?

Reply:
Existing code https://github.com/Workday/canvas-kit/blob/master/modules/react/common/lib/CanvasProvider.tsx#L77

Reply:
It looks like a Prettier reformat. `'action'` was removed from the array and that shortened the line enough.